geode-docs/developing/querying_basics/restrictions_and_unsupported_features.html.md.erb

```diff
@@ -30,6 +30,6 @@ At a high level, <%=vars.product_name%> does not support the following querying
 
 -   You cannot create an index on fields using Set/List types (Collection types) that are not comparable. The OQL index implementation expects fields to be Comparable. To workaround this, you can create a custom Collection type that implements Comparable.
 -   ORDER BY is only supported with DISTINCT queries.
-
+-   OQL queries aren't pre-validated for references to nonexistent methods and attributes.
```

Comment:
I have commented on the ticket with my thoughts. 
What could be added here is, saying OQL query engine determines the field or methods on the given object (region keys and values) during query execution time.

Reply:
I will propose a rewrite tomorrow that uses @agingade's verbiage.

Reply:
[My suggested revision, adding some of Anil's explanation from the JIRA comments for context, and expanding contractions to whole words for better translation/localization]
In the Geode model, attributes and methods are resolved at runtime during query execution. This means that OQL queries are not pre-validated for references to nonexistent methods and attributes.

Reply:
Thank you both for comments and proposals. I have updated PR based on the @davebarnes97 proposal.